srcpkgs/whdd/patches/gnu-source.patch

```diff
@@ -0,0 +1,11 @@
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -5,7 +5,7 @@
+ option(STATIC "Build static binaries" OFF)
+ option(CLI "Build whdd-cli" OFF)
+ 
+-set(CMAKE_C_FLAGS "-std=gnu99 -pthread -Wall -Wextra -Wno-missing-field-initializers ${CFLAGS}")
++set(CMAKE_C_FLAGS "-D_GNU_SOURCE -std=gnu99 -pthread -Wall -Wextra -Wno-missing-field-initializers ${CFLAGS}")
```

Comment:
Isn't setting `CFLAGS="-D_GNU_SOURCE"` in the template enough?

Reply:
yes, it's enough

Reply:
**UPD 2**: putting the patch back fixed it

Reply:
Did you take the patch from https://github.com/whdd/whdd/pull/12 ? This really should be upstreamed :/

Reply:
I think that patching 1 file is better than patching 3. The author isn't interested in upstreaming the patch.
Despite not being actively developed, I find `whdd` very useful, because it's the only one program with Victoria-like (https://hdd.by/victoria/) disk scanning visualization.